opencti-platform/opencti-graphql/src/modules/notification/notification-resolver.ts

```diff
@@ -3,29 +3,37 @@ import type { Resolvers } from '../../generated/graphql';
 import {
   addDigestTrigger,
   addLiveTrigger,
+  myInstanceTriggersFind,
   myNotificationsFind,
+  myTriggersFind,
   myUnreadNotificationsCount,
   notificationDelete,
   notificationEditRead,
   notificationGet,
   notificationsFind,
+  resolvedInstanceFiltersGet,
   triggerDelete,
   triggerEdit,
   triggerGet,
   triggersFind,
-  myTriggersFind,
   triggersGet,
 } from './notification-domain';
 import { pubSubAsyncIterator } from '../../database/redis';
 import { BUS_TOPICS } from '../../config/conf';
-import { ENTITY_TYPE_NOTIFICATION, NOTIFICATION_NUMBER } from './notification-types';
+import {
+  BasicStoreEntityLiveTrigger,
+  BasicStoreEntityTrigger,
+  ENTITY_TYPE_NOTIFICATION,
+  NOTIFICATION_NUMBER
+} from './notification-types';
 
 const notificationResolvers: Resolvers = {
   Query: {
     // Triggers
     trigger: (_, { id }, context) => triggerGet(context, context.user, id),
     triggers: (_, args, context) => triggersFind(context, context.user, args),
     myTriggers: (_, args, context) => myTriggersFind(context, context.user, args),
+    myInstanceTriggers: (_, { instanceId }, context) => myInstanceTriggersFind(context, context.user, instanceId),
```

Comment:
Should be unified in triggers and mytriggers

Reply:
done